x-pack/legacy/plugins/apm/public/components/shared/Stacktrace/CauseStacktrace.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+import { i18n } from '@kbn/i18n';
+import { EuiAccordion, EuiTitle } from '@elastic/eui';
+import { px, unit } from '../../../style/variables';
+import { Stacktrace } from '.';
+import { IStackframe } from '../../../../typings/es_schemas/raw/fields/Stackframe';
+
+// @ts-ignore Styled Components has trouble inferring the types of the default props here.
+const Accordion = styled(EuiAccordion)`
+  border-top: ${theme.euiBorderThin};
+`;
+
+const CausedByContainer = styled('h5')`
+  padding: ${theme.spacerSizes.s} 0;
+`;
+
+const CausedByHeading = styled('span')`
+  color: ${theme.textColors.subdued};
+  display: block;
+  font-size: ${theme.euiFontSizeXS};
+  font-weight: ${theme.euiFontWeightBold};
+  text-transform: uppercase;
+`;
```

Comment:
Is it necessary to set this up as a styled component. We should be able to leverage `EuiText` heading six (uppercase) with a subdued color. Thoughts?

Reply:
I set this up so the whole thing (`CausedbyContainer`) is an `h5` with a couple spans in it, since having an `h6` directly followed by an `h5` would make sense and would probably fail a11y checks.

I think I can still do what you're suggesting using `EuiText` though so I'll try it out. It might still need to be a styled component for the block display though.

Reply:
So an `EuiText` doesn't let you do `text-transform`, and an `EuiTitle` doesn't let you do `color`. I think I'm going to leave this as-is for now unless you still want it changed.

Reply:
@smith I looked into it myself and yes, the EuiText and EuiTitle supports neither case. I think I'll bring this up with the team whether either should support it. Let's keep as is 👍